src/main/java/org/paymentsds/mpesa/Client.java

```diff
@@ -0,0 +1,99 @@
+package org.paymentsds.mpesa;
+
+public class Client {
+    private final String apiKey;
+    private final String publicKey;
+    private final String serviceProviderCode;
+    private final String initiatorIdentifier;
+    private final Environment environment;
+    private final String host;
+    private final String securityCredential;
+
+    private Client(
+            String apiKey,
+            String publicKey,
+            String serviceProviderCode,
+            String initiatorIdentifier,
+            Environment environment,
```

Comment:
I think we can remove this property, we just need it to fill the host name

Reply:
Sounds good. I'll remove the Environment property